docs/UserGuide.md

```diff
@@ -356,47 +348,99 @@ A lesson **must** have the following fields: a start date, a time range, a lesso
-as well as specify minimum/maximum lesson durations so that accidents like mistyping a 10-minute lesson won't happen.
-Additionally, we will allow you to toggle these checks on and off, just in case you don't want them.
-Do look forward to these features!
-
-<div markdown="block" class="alert alert-info">
-**:information_source: Note:**<br>
-* The lesson's rate refers to the fee of the lesson per hour.
-This rate will be used in the calculation of fees due after each lesson.
-
-* A lesson can be identified by the index number shown in the lesson list of the student.
-</div>
-
-<br>
+   
+In the future, you can also customise your own recurrence rule for your recurring lessons.
+
+A lesson can be identified by the index number shown in the lesson list of the student. The lesson index specified for commands must be a valid index number shown in the lesson list of the student.
+
+<table id="student-param-table">
+    <thead>
+        <tr>
+            <th style="text-align:center; padding: 10px">Category</th>
+            <th style="text-align:center">Field</th>
+            <th style="text-align:center">Prefix</th>
+            <th style="text-align:center">Constraints</th>
+            <th style="text-align:center">Examples</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan=4>Essential</td>
+            <td>Start date</td>
+            <td><code>date/</code></td>
+            <td>
+              <ul>
+                <li>Case-insensitive.</li>
+                <li>Formatted as <code>dd MMM yyyy</code></li>
+              </ul>
+            </td>
+            <td><code>date/12 jul 2020</code></td>
+        </tr>
+        <tr>
+            <td>Subject</td>
+            <td><code>subject/</code></td>
+            <td>Should only contain alphanumeric characters and spaces</td>
+            <td><code>subject/Social studies</code></td>
+        </tr>
+        <tr>
+            <td>Time range</td>
+            <td><code>time/</code></td>
+            <td><ul>
+            <li>Formatted as <code>HHmm-HHmm</code></li>
+            <li>Must be between 8am and 10pm, inclusive</li>
+            </ul></td>
+            <td><code>time/1730-1830</code></td>
+        </tr>
+        <tr>
+            <td>Lesson rate</td>
+            <td><code>rates/</code></td>
+            <td><ul>
+              <li>Fee of the lesson per hour. This rate will be used in the calculation of fees due after each lesson.</li>
+              <li>Refer to <a href="#monetary-fields">Monetary fields</a> for more details.</li>
+            </ul></td>
+            <td><code>rates/50</code></td>
+        </tr>
+        <tr>
+        <td rowspan=5>Optional</td>
+            <td>Recurrence flag</td>
+            <td><code>recurring/</code></td>
+            <td><ul>
+              <li>Indicates whether the lesson is recurring</li>
+              <li>Optional parameter: end date of the recurrence</li>
```

Comment:
End date is the parameter value
```suggestion
              <li>Optional value: end date of the recurrence</li>
```

Reply:
By the command format notes under Features, the parameter is the value

Reply:
icic ok